app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -79,13 +79,16 @@ def update
     param :description, String, :desc => N_("Description for the new published content view version")
     param :force_yum_metadata_regeneration, :bool, :desc => N_("Force yum metadata regeneration on the repositories " \
                                                            "in the content view version")
-    param :major, :number, :desc => N_("Override the major version number"), :required => false
-    param :minor, :number, :desc => N_("Override the minor version number"), :required => false
+    param :major, :number, :desc => N_("Override the major version number")
+    param :minor, :number, :desc => N_("Override the minor version number")
+    param :repos_units, Hash, :desc => N_("list of repositories and units to publish")
```

Comment:
You should be able to describe this hash in more detail as part of the param definition (showing the structure)